File: turtle_keyboard/src/node.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

geometry_msgs::Twist output;

void callback(const geometry_msgs::Twist::ConstPtr &input) {
	output = *input;
}

int main(int argc, char **argv) {
	//Initiate ROS
	ros::init(argc, argv, "node");  // 노드명 초기화
	ros::NodeHandle nh;
	ros::Subscriber sub=nh.subscribe("/cmd_vel", 100, callback);
	ros::Publisher pub=nh.advertise<geometry_msgs::Twist>("/turtle1/cmd_vel", 100);

	ros::Rate loop_rate(10);

	while(ros::ok()) {		
		pub.publish(output);
		loop_rate.sleep();
		ros::spinOnce();
	}
	return 0;
}

File: turtle_keyboard_py/scripts/node.py
#!/usr/bin/env python

import rospy
from geometry_msgs.msg import Twist

DIRECTION = Twist()

def callback(input):
    global DIRECTION
    DIRECTION = input

def key_pub():
    rospy.init_node('key_pub', anonymous=True)
    sub = rospy.Subscriber('cmd_vel', Twist, callback)
    pub = rospy.Publisher('turtle1/cmd_vel', Twist, queue_size=100)
    rate = rospy.Rate(10) # 10hz

    global DIRECTION

    while not rospy.is_shutdown():
        pub.publish(DIRECTION)
        rate.sleep()

if __name__ == '__main__':
    try:
        key_pub()
    except rospy.ROSInterruptException:
        pass